helper/fetch-release

```diff
@@ -14,15 +14,15 @@ fi
 TMPDIR=$(mktemp -d)
 trap "rm -rf $TMPDIR" EXIT
 
-GPG_HOME=$TMPDIR/gpg-home
+export GPG_HOME=$TMPDIR/gpg-home
```

Comment:
isn't this variable called `GNUPGHOME`?

Reply:
Aww, not my proudest moment. Thanks for catching this.
I quickly drafted that commit a while a ago and then just cherry-picked it without proper checking.

Reply:
I only noticed this because the commit message said that we would follow the same approach with the fetcher scripts, but wasn't able to find GPG_HOME.